content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,219 @@
+    10.1.158.192/26 via 9.111.255.129 dev tunl0 proto bird onlink
+    blackhole 10.1.198.128/26 proto bird
+    {{< /text >}}
+    {{< text bash >}}
+    root@gyliu-icp-2:~# ip route | grep bird
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    blackhole 10.1.158.192/26  proto bird
+    {{< /text >}}
+    {{< text bash >}}
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    root@gyliu-icp-3:~# ip route | grep bird
+    blackhole 10.1.43.0/26  proto bird
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  You can see there are three IP routers total for those three nodes in cluster-1.
+
+    {{< text bash >}}
+    10.1.158.192/26 via 9.111.255.129 dev tunl0  proto bird onlink
+    10.1.198.128/26 via 9.111.255.21 dev tunl0  proto bird onlink
+    10.1.43.0/26 via 9.111.255.29 dev tunl0  proto bird onlink
+    {{< /text >}}
+
+1.  Add those three IP routers to all nodes in cluster-2 by the command as follows.
+
+    {{< text bash >}}
+    $ ip route add 10.1.158.192/26 via 9.111.255.129 
+    $ ip route add 10.1.198.128/26 via 9.111.255.21
+    $ ip route add 10.1.43.0/26 via 9.111.255.29
+    {{< /text >}}
+
+1.  You can use same steps to add all IP routers from cluster-2 tocluster-1, as well. After the configuration
+    finished, you will see that the pods in those two different clusters can communication with each other.
+
+1.  You can verify it by pinging pod IP in cluster-2 from cluster-1. The following is a pod from cluster-2 with
+    pod IP as 20.1.47.150.
+
+    {{< text bash >}}
+    root@gyliu-ubuntu-1:~/cluster# kubectl get pods -owide  -n kube-system | grep platform-ui
+    platform-ui-lqccp                                             1/1       Running     0          3d        20.1.47.150     9.111.255.77
+    {{< /text >}}
+
+1.  In one of cluster-1 node, ping this pod IP, it should succeed.
+
+    {{< text bash >}}
+    [root@gyliu-icp-1 mc]# ping 20.1.47.150
+    PING 20.1.47.150 (20.1.47.150) 56(84) bytes of data.
+    64 bytes from 20.1.47.150: icmp_seq=1 ttl=63 time=0.759 ms
+    {{< /text >}}
+
+The above configuration actually configured full IP route mesh across all nodes in the two IBM Cloud Private Clusters,
+which enabled Pod communication cross clusters.
+
+## Install the Istio for multi cluster
+
+You can follow steps [here](/docs/setup/kubernetes/multicluster-install/) to install and config local Istio control plane
+and Istio remote on cluster-1 and cluster-2.
+
+In this example, I was using cluster-1 as local Istio control plane and cluster-2 as Istio remote.
+
+## Deploy Bookinfo Example Across Clusters
+
+__NOTE__: The following example is enabling [sidecar automatic injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
+
+1.  Install bookinfo on the cluster-1. Remove the `reviews-v3` deployment to deploy on remote:
```

Comment:
all instances of cluster-1 and cluster-2 should have backticks

Reply:
Done